### DISCUSSION TITLE
GPTQ-INT8 quantization details: W8A16 or W8A8?

### DISCUSSION THREAD

**Discussion by molereddy** [QwenLM/Qwen3#1058] [General]
Can you clarify on if the 8-bit GPTQ quantized models use 8-bit activations or 16-bit? Model link: https://huggingface.co/Qwen/Qwen2.5-72B-Instruct-GPTQ-Int8

**Comment by jklj077** [comment]
GPTQ is weight-quantization. GPTQ-INT8 is 8bit weight + fp16 scales + fp16 zeros (with fp16 activation).
